linux/open_vm_tools/install_ovt_from_source.yml

```diff
@@ -8,14 +8,20 @@
     ovt_install_prefix: "{{ (guest_os_ansible_system == 'linux') | ternary('/usr', '/usr/local') }}"
     ovt_install_lib: |-
       {%- if guest_os_ansible_system == 'freebsd' -%}/usr/local/lib
-      {%- elif guest_os_family == 'RedHat' -%}/usr/lib64
+      {%- elif guest_os_family == 'RedHat' and guest_os_ansible_distribution not in ['FusionOS', 'Kylin Linux Advanced Server'] -%}/usr/lib64
       {%- else -%}/usr/lib
       {%- endif -%}
     ovt_build_config_file_tmpl: "{{ guest_os_ansible_system }}_ovt_build_config.tmpl"
     ovt_build_config_file_path: "{{ current_test_log_folder }}/ovt_build_config.yml"
     ovt_check_config_file_tmpl: "{{ guest_os_ansible_system }}_ovt_check_config.tmpl"
     ovt_check_config_file_path: "{{ current_test_log_folder }}/ovt_check_config.yml"
 
+- name: "Ensure directory exists for build config file"
+  ansible.builtin.file:
+    path: "{{ ovt_build_config_file_path | dirname }}"
+    state: directory
+    mode: "0755"
```

Comment:
The current_test_log_folder is created at test setup, so here doesn't need to create it again.

Reply:
You can add a var to force to create the log folder at test setup in https://github.com/vmware/ansible-vsphere-gos-validation/blob/main/linux/open_vm_tools/pre_ovt_install.yml#L11

```
        - name: "Test setup"
          include_tasks: ../setup/test_setup.yml
          vars:
            create_test_log_folder: true
```